src/display/text_layer.js

```diff
@@ -63,6 +63,7 @@ var renderTextLayer = (function renderTextLayerClosure() {
     // Initialize all used properties to keep the caches monomorphic.
     var textDiv = document.createElement('div');
     var textDivProperties = {
+      style: '',
```

Comment:
Can't we initialize this to `null` to avoid allocating an unnecessary string? It appears to be only used below, and before that it's set already.